test/e2e/autoscale_test.go

```diff
@@ -126,6 +128,53 @@ func generateTraffic(ctx *testContext, concurrency int, duration time.Duration,
 	return nil
 }
 
+func generateTrafficAtFixedRPS(ctx *testContext, rps int, duration time.Duration, stopChan chan struct{}) error {
+	var (
+		totalRequests      int32
+		successfulRequests int32
+	)
+
+	rate := vegeta.Rate{Freq: rps, Per: time.Second}
+	target := vegeta.Target{
+		Method: "GET",
+		URL:    fmt.Sprintf("http://%s?sleep=100", ctx.domain),
```

Comment:
Don't you need to do the resolution dance on whether or not `resolvabledomain` is set etc?

Reply:
Didn't know that we have `resolvabledomain` flag. Done.